x-pack/plugins/maps/common/elasticsearch_util/elasticsearch_geo_utils.test.js

```diff
@@ -477,52 +477,30 @@ describe('createExtentFilter', () => {
         minLat: 35,
         minLon: -89,
       };
-      const filter = createExtentFilter(mapExtent, geoFieldName, 'geo_shape');
+      const filter = createExtentFilter(mapExtent, geoFieldName);
```

Comment:
Can you remove test case "should return elasticsearch geo_shape filter". Its now the same as the test case on 393.